l10n_eu_product_adr/models/product_template.py

```diff
@@ -1,118 +1,43 @@
 # Copyright 2019 Iryna Vyshnevska (Camptocamp)
+# Copyright 2021 Opener B.V. <stefan@opener.amsterdam>
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
-from odoo import _, api, fields, models
-
-LABELS_SELECTION = [
-    ("1", "2"),
-    ("2", "2.1"),
-    ("3", "2.2"),
-    ("4", "3"),
-    ("5", "4.1"),
-    ("6", "4.2"),
-    ("7", "4.3"),
-    ("8", "5.1"),
-    ("9", "5.2"),
-    ("10", "8"),
-    ("11", "9"),
-    ("12", "9A"),
-]
-
-TRANSPORT_CATEGORY = [("1", "0"), ("2", "1"), ("3", "2"), ("4", "3"), ("5", "4")]
+from odoo import api, fields, models
 
 
 class ProductTemplate(models.Model):
     _inherit = "product.template"
 
-    is_dangerous = fields.Boolean(string="Dangerous product")
-    dangerous_class_id = fields.Many2one(
-        "product.dangerous.class", string="Dangerous class"
+    is_dangerous = fields.Boolean(help="This product belongs to a dangerous class")
+    adr_goods_id = fields.Many2one("adr.goods", "Dangerous Goods")
+    adr_class_id = fields.Many2one(
+        "adr.class", related="adr_goods_id.class_id", readonly=True
```

Comment:
Since the dangerous attributes can vary from a variant to another, we moved those fields from `product.template` to `product.variant`.
see this PR https://github.com/OCA/community-data-files/pull/112

Could you please do the same?

Reply:
Alright, I will take care of that.

Reply:
See new commit `[RFR] l10n_eu_product_adr: allow different adr goods per variant`